scripts/oca_dependencies.txt

```diff
@@ -0,0 +1,2 @@
+social https://github.com/OCA/social 8.0
+server-tools https://github.com/OCA/server-tools 8.0
```

Comment:
What do you mean by default module? This is default syntax of the OCA `oca_dependencies.txt`.